### PR TITLE
Updated Data-Drive Design content

### PR DESCRIPTION
**Pull request by moingshaikh**
Added detailed introduction to Data Driven Design. What, Why, and How of the subject and also added tools and resources for the same.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/user-experience-research/data-driven-design/index.md [hunk 1]
```diff
@@ -3,11 +3,49 @@ title: Data Driven Design
...
+
+Customers revealing real answers to these questions can be real gem for you. If you get to know the answers to them questions, you are half way through your data driven design.
+
+**3)	A/B Test**
+
+Another popular method is to conduct A/B Test on your website or app. An A/B test is simply a test of two different version of the same element (such as banner, call to action button, headline, price, color etc.) on your website.
+
+An effective A/B test can tell you which version of your tested element is more popular among customers. 
+
+For example: If you want to redesign your home-page using data driven design approach, you can conduct A/B test with two different design of the home-page and directing a few of users on 1st version of the design, while directing other users on the 2nd version and thus noticing which version retains or converts more users during a specific test duration.
```
Comment (moT01) [inline guide/english/user-experience-research/data-driven-design/index.md]:
```suggestion
For example: If you want to redesign your home-page using data driven design approach, you can conduct A/B test with two different designs of the home-page and directing a few of users on first version of the design, while directing other users on the second version. With a test like this, you can find out which version retains or converts more users during a specific test duration. 
```

**Review by moT01 — DISMISSED** [review]
This looks real good @moingshaikh - but has quite a few typos. I have suggested changes on the typos and reworded a few things to make it sound a little better. It would be great if you could fix those and we can merge this. Thanks.

File: guide/english/user-experience-research/data-driven-design/index.md [hunk 2]
```diff
@@ -3,11 +3,49 @@ title: Data Driven Design
 ---
 ## Data Driven Design
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/user-experience-research/data-driven-design/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+**What is Data Driven Design?**
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+Data-driven design is a data-based, strategic design approch that aims to create web/app/software or any other digital product on the basis of available user-data gathered using a number of ways such as user interviews, analytics interpretation, surveys, research and so on.
```
[the remark this answers is not in the document]
Comment (moingshaikh) [reply guide/english/user-experience-research/data-driven-design/index.md]: Thank you, @moT01 for your time and review and suggestions; I'll work on them and let you know soon. I would also like to apologise for the typos.

[the remark this answers is not in the document]
Comment (moT01) [reply guide/english/user-experience-research/data-driven-design/index.md]: Thank you - if you want, you can add them directly on github by clicking the `commit suggestion` button

**Review by jonathan-grah — APPROVED** [review]
### LGTM 👍

**Review by QuincyLarson — APPROVED** [review]
Solid work, @moingshaikh. Thanks again for your contribution.

**Pull request merged**